src/app/carbon-estimator-form/carbon-estimator-form.component.ts

```diff
@@ -187,6 +188,8 @@ export class CarbonEstimatorFormComponent implements OnInit {
 
   public resetForm() {
     this.estimatorForm.reset();
+    this.validationErrors = [];
```

Comment:
Only a couple of lines but maybe put these into a `resetErrors()` method if they always come together?